Makefile

```diff
@@ -31,40 +32,59 @@ all: docker
 gobuild:
 	./scripts/build false
 
+# create output directories
+.out-stamp:
```

Comment:
This is not a `PHONY` target on purpose?

Reply:
Yep! It creates a file called `.out-stamp` that tells `make` not to rerun this step.